src/tasks/runstart.ts

```diff
@@ -145,6 +154,15 @@ export const RunStartQuest: Quest = {
       },
       limit: { tries: 1 },
     },
+    {
+      name: "Set up Sweatsuit",
+      ready: () => have($item`tiny stillsuit`),
+      completed: () => familiarEquippedEquipment(bestStillsuitFamiliar) === $item`tiny stillsuit`,
+      do: (): void => {
+        equip(bestStillsuitFamiliar, $item`tiny stillsuit`);
+      },
+      limit: { tries: 1 },
+    },
```

Comment:
It's hard to imagine a Guy that neither has a volleyball nor mosquito, but we should do our due dilligence to ensure that `bestStillsuitFamiliar` is not `$familiar.none` after we update the variable above

Reply:
You cannot not have a mosquito. It's mandatory to ascend once, which is mandatory to enter CS.